app.js

```
@@ -1,33 +1,23 @@
 const express = require('express');
 const socketIO = require('socket.io');
-const mqtt = require('mqtt');
-const uuid = require('uuid');
+const http = require('http');
```

Biome

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)